### PR TITLE
Azure pipeline:| We don't want to use a private feed in a public repository

### PR DESCRIPTION
**Pull request by mmsmits**

### CODE REVIEW

File: build/azure-pipelines.yml [hunk 1]
```diff
@@ -34,8 +34,9 @@ stages:
         propsFile: 'Firely.Fhir.Packages.props'
         packageArtifacts: true
         restoreDependencies: true
-        nuGetServiceConnections: GitHubPackageGetFeed
-        nuGetSources: --source https://nuget.pkg.github.com/FirelyTeam/index.json
+        # we shouldn't use a private feed since we are public repo
```
Comment (louisledev) [inline build/azure-pipelines.yml]: Looking at the template, I see some inconsistency:
```
nuGetServiceConnections: #required when restoreDependies = true
nuGetSources: #required when restoreDependies = true
 ```
and digging further, in https://github.com/FirelyTeam/azure-pipeline-templates/blob/193a8796cd4b68e784a6f33a8699b01036383a1f/restore.yml: 
```
  - task: NuGetAuthenticate@1
    inputs:
      nuGetServiceConnections: ${{ parameters.nuGetServiceConnections }}
```
Should we make a check on the nuGetServiceConnections not being empty?

**Review by louisledev — CHANGES_REQUESTED** [review]

File: build/azure-pipelines.yml (hunk 1 above)
Comment (mmsmits) [reply build/azure-pipelines.yml]: I think that's at least nice to do, although it currently works without it being present. 
Also, and `nuGetSources` should just not be marked as required

Comment (mmsmits) [reply build/azure-pipelines.yml]: See: https://github.com/FirelyTeam/azure-pipeline-templates/pull/25

File: build/templates/build.yml [hunk 2]
```diff
@@ -7,8 +7,8 @@ parameters:
   useVersionSuffix: true # if true, the build number will be added as or to the existing version suffix
   propsFile: '' # used for versionSuffix, if not empty, the versionSuffix will be read from the props file
   restoreDependencies: false
-  nuGetServiceConnections: #required when restoreDependies = true
-  nuGetSources: #required when restoreDependies = true
+  nuGetServiceConnections: #service connetions to the additional nuget feeds
```
Comment (louisledev) [inline build/templates/build.yml]:
```suggestion
  nuGetServiceConnections: #service connections to the additional nuget feeds
```

Comment (louisledev) [reply build/templates/build.yml]: Also, just thinking, how do you provide multiple connections? comma-delimited?

**Review by louisledev — CHANGES_REQUESTED** [review]

**Review by louisledev — APPROVED** [review]

**Pull request merged**